Gemfile

```diff
@@ -0,0 +1,100 @@
+source "https://rubygems.org"
+
+git_source(:github) {|repo_name| "https://github.com/#{repo_name}.git" }
+
+# ruby version
+ruby '2.7.1'
+
+#
+# Rails essentials
+#
+
+# The application framework. https://github.com/rails/rails
+gem 'rails', '~> 6.0', '>= 6.0.3.4'
+# Use sqlite3 as the database for Active Record. https://www.sqlite.org/index.html
+gem 'sqlite3', '~> 1.4', '>= 1.4.2'
+# Use Puma as the app server. https://github.com/puma/puma
+gem 'puma', '~> 5.0', '>= 5.0.2'
+# Use Uglifier as compressor for JavaScript assets. https://github.com/lautis/uglifier
+gem 'uglifier', '~> 4.2'
+# See https://github.com/rails/execjs#readme for more supported runtimes
+gem 'mini_racer', '~> 0.3.1'
```

Comment:
Is this still required in Rails 6? Is it included in the default Gemfile that is created for a new Rails 6 project? If I understood correctly, Node.js is required now anyways, so `execjs` could use that instead of `mini_racer`

Reply:
The default Rails 6 Gemfile does not contain `mini_racer`, so I've removed it.